rudder-lang/src/technique.rs

```diff
@@ -220,8 +221,12 @@ pub struct MethodCall {
     condition: String,
     method_name: String,
     component: Option<String>,
+    #[serde(default = "generate_id")] // >=7.0
     id : String,
 }
+fn generate_id () -> String {
```

Comment:
could you run `cargo fmt` on your project?

Reply:
done ! i did not know this pro-tip